### PR TITLE
Able to set description on creation card via API.

### PR DESCRIPTION
**Pull request by a11exandru**
Signed-off-by: Alexandru Puiu <alexpuiu20@yahoo.com>

* Resolves: #1138 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: lib/Controller/CardApiController.php [hunk 1]
```diff
@@ -70,11 +72,12 @@ public function get() {
 	 * @params $title
 	 * @params $type
 	 * @params $order
+	 * @params $description
 	 *
 	 * Get a specific card.
 	 */
-	public function create($title, $type = 'plain', $order = 999) {
-		$card = $this->cardService->create($title, $this->request->getParam('stackId'), $type, $order, $this->userId);
+	public function create($title, $type = 'plain', $order = 999, $description) {
```
Comment (juliushaertl) [inline lib/Controller/CardApiController.php]: This should have an empty default value

File: lib/Service/CardService.php [hunk 2]
```diff
@@ -594,7 +597,7 @@ public function assignUser($cardId, $userId) {
 			/* Notifyuser about the card assignment */
 			$this->notificationHelper->sendCardAssigned($card, $userId);
 		}
-
+
```
Comment (juliushaertl) [inline lib/Service/CardService.php]: Please remove the tabs in empty lines

File: lib/Controller/CardApiController.php (hunk 1 above)
Comment (a11exandru) [reply lib/Controller/CardApiController.php]: Ok, it's done!

File: lib/Service/CardService.php (hunk 2 above)
Comment (a11exandru) [reply lib/Service/CardService.php]: Done

**Pull request merged**